ibm-qradar-edr/info.json

```diff
@@ -0,0 +1,823 @@
+{
+  "name": "ibm-qradar-edr",
+  "label": "IBM QRadar EDR",
```

Comment:
The connector name should be IBM Security QRadar EDR.
Please update the git repo name also

Reply:
Updated